docs/basics.rst

```diff
@@ -16,7 +16,7 @@ subsequently be ready to accept form submission input are:
 - Render the form.
 
 Once the form is rendered, a user will interact with the form in his
-browser, and some point, he will submit it.  When the user submits the
+browser, and some point, they will submit it.  When the user submits the
```

Comment:
Maybe better to drop the pronoun altogether here?  E.g.

    browser, and at some point, will submit it.

Note also the `his` at the end of the previous line.